wsd/COOLWSD.hpp

```diff
@@ -350,6 +351,11 @@ class COOLWSD final : public Poco::Util::ServerApplication,
         return Util::encodeId(NextConnectionId++, 3);
     }
 
+    static std::string getHardwareResourceWarning()
```

Comment:
`const std::string&` ?

Reply:
Thank you for the suggestion. 
I have updated the getHardwareResourceWarning method to return a constant reference